pypesto/__init__.py

```diff
@@ -17,6 +17,7 @@
     OptimizerHistory,
     AmiciObjective,
     Objective,
+    Priors,
```

Comment:
Note: Effectively this is a `LogPrior`. However, also other statistical inference tools are ignorant of this fact. Thus, calling it `Prior` and not e.g. `LogPrior` would be fine for me. Opinions?

Reply:
Good catch, will change that!

Reply:
On the other hand: This is also a _negative_ `LogPrior` so how about that?

Reply:
I am really not sure what is best. I would tend to calling it "Prior", but in the docstring make a note/warning that it expects everything on log-scale. I think this is what e.g. pymc3 does as well (except not such a proper warning ...).

Reply:
> On the other hand: This is also a _negative_ `LogPrior` so how about that?

True. I would rather say the negative needs to be highlighted. This is why we called it "Objective" also, not "Likelihood" or sth. :thinking: negative prior makes no sense at all, so maybe indeed `NegLogPrior`?